src/components/readArticle.jsx

```diff
@@ -25,6 +29,16 @@ class ReadArticle extends Component {
     readArticles(params.slug);
   }
 
+  likeArts = async (slug) => {
```

Comment:
You have added async here and there is nowhere inside the function you call await...